packages/grant-explorer/src/features/round/ViewCartPage.tsx

```diff
@@ -98,6 +98,7 @@ export default function ViewCart() {
       ? { addressOrName: address }
       : { addressOrName: address, token: selectedPayoutToken.address };
 
+  // @ts-expect-error Temp until viem
```

Comment:
what's the error we can have here?

Reply:
@ts-expect-error silences the typescript error on the next line. unlike ts-ignore, it will error out if there is no error on the next line, making sure that you remove the ts-expect-error once it's not necessary anymore.
In this case, you can see the error by removing the @ts-expect-error line. 
The error in this case comes from wagmi requiring addresses to be of type ``0x${string}`` (a hexadecimal string). In viem, you can import this type (called `Hex`), however I couldn't find the import in our current version of wagmi. We silence the error here for now, and it will alert us to remove the expect-error once we adopt viem.